### DISCUSSION TITLE
What should I do if I (accidentally) remove the `packer_compiled.lua` file

### DISCUSSION THREAD

**Discussion by amirhhashemi** [wbthomason/packer.nvim#986] [Q&A]
I accidentally removed the `packer_compiled.lua` file. After that, I was getting wired errors on startup and I didn't have access to the packer commands. Google didn't help me and eventually, I decided to completely uninstall and install neovim. That may sound like a weird thing to do in this situation but I was exhausted and that was the first thing that came to my mind. Anyway, I removed the `~/.local/share/nvim`, `~/.cache/nvim`, and the neovim binary (I didn't install neovim with my package manager). Then I downloaded the neovim binary from github and put it in my PATH, installed packer, and ran neovim without any problem. Then I ran `PackerSync` and expected packer to re-generate the `packer_compiled` file but that didn't happen. It just shows me a blank floating window:

![image](https://user-images.githubusercontent.com/87268103/181801737-84d2f120-37db-444d-b4c3-58091c3b3f3a.png)

That actually has happened to me before. If I just move my configs several times it will magically resolve but I want to know what should I do if that happens again. 

I'm using Arch (BTW) and neovim nightly.

**Comment by bryant-the-coder** [comment]
just run `:PackerSync` again

**Reply by amirhhashemi** [reply]
For the 10032095th time?
